metadata-models/src/main/pegasus/com/linkedin/dataquality/DimensionScore.pdl

```diff
@@ -0,0 +1,52 @@
+namespace com.linkedin.dataquality
+
+/**
+ * Record to capture Dimension Score information
+ */
+record DimensionScore {
+  /**
+  * dimensionName attribute to capture predefined dimension names.
+  * Required field if this record is initialized
+  */
+  dimensionName: enum DimensionName {
+    ACCURACY
+    COMPLETENESS
+    CONSISTENCY
+    INTEGRITY
+    SEMANTIC_CORRECTNESS
+    UNIQUENESS
+    VALIDITY
+    TIMELINESS
+    RELIABILITY
+    DUPLICATION
+    PRECISION
+  }
+
+  /**
+  * currentScore attribute to capture current score for a specific dimension
+  * Required field if this record is initialized
+  */
+  currentScore: string
+
+  /**
+  * historicalWeightedScore attribute to capture historical weighted score for a specific dimension
+  * Optional field. Value will be populated, if available
+  */
+  historicalWeightedScore: optional string
+
+  /**
+  * scoreType attribute to capture score type for a specific score
+  * Required field if this record is initialized
+  */
+  scoreType: enum ScoreType {
+    PERCENTAGE
+    NUMERICAL_VALUE
```

Comment:
I am by no means an expert on the subject so forgive my ignorance, but how do you map a numerical value to the "accuracy" and "completeness"?  Or would a boolean type be a good addition here?

Reply:
+1 - Also interested to understand this

Reply:
For "accuracy" and "completeness" dimensions score type would be PERCENTAGE. 
As ScoreType is an enum it can take either PERCENTAGE or NUMERICAL_VALUE. 
 
ScoreType defined to be either PERCENTAGE and NUMERICAL_VALUE, if other values are expected than these two, then let us know if you would prefer adding dynamic list for ScoreType as well.